rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelector.xtend

```diff
@@ -74,12 +95,11 @@ class TestStepSelector {
 		dragSourceListener.viewer = viewer
 		viewer.contentProvider = amlInjectorProvider.get.getInstance(TestStepSelectorTreeContentProvider)
 		viewer.labelProvider = labelProvider
-		
 		Job.jobManager.addJobChangeListener(new JobChangeAdapter {
 			override done(IJobChangeEvent event) {
 				if (event.job.name.equals("Building workspace")) {
 					logger.info("Building workspace completed. Trigger update TestStepSelector")
-					broker.post(TestStepSelector.SELECTOR_UPDATE_MODEL_AND_VIEW, null)
+					broker.post(TestStepSelector.SELECTOR_UPDATE_VIEW, null)
```

Comment:
We need to call the listener right after start otherwhile the selector will stay empty. I changed the code to use the current activePage